### PR TITLE
Fix feature labels for docs.rs

### PR DESCRIPTION
**Pull request by urschrei**
Following up on https://github.com/georust/geojson/pull/136#discussion_r483121450

See here: https://stackoverflow.com/a/61417700/416626

### CODE REVIEW

File: src/conversion/from_geo_types.rs [hunk 1]
```diff
@@ -1,3 +1,7 @@
+// only enables the `doc_cfg` feature when
+// the `docsrs` configuration attribute is defined
+#[cfg_attr(docsrs, feature(doc_cfg))]
```
Comment (michaelkirk) [inline src/conversion/from_geo_types.rs:3]: No biggie, so take it or leave it, but can we do this in a single place, like at src/lib.rs or does it need to be duplicated around?

**Review by michaelkirk — APPROVED** [review]
Thanks! Feel free to merge after addressing or declining my optional feedback.

File: src/conversion/from_geo_types.rs (hunk 1 above)
Comment (urschrei) [reply src/conversion/from_geo_types.rs:3]: I have no idea, but that would be better. I'll ask elsewhere…

Comment (michaelkirk) [reply src/conversion/from_geo_types.rs:3]: Tokio does it in the crate root - so seems like it should work.

https://github.com/tokio-rs/tokio/blob/master/tokio/src/lib.rs#L18

**Pull request merged**